contracts/reputationMiningCycle/ReputationMiningCycleCommon.sol

```diff
@@ -115,4 +124,64 @@ contract ReputationMiningCycleCommon is ReputationMiningCycleStorage, PatriciaTr
         y := add(y, mul(256, gt(arg, 0x8000000000000000000000000000000000000000000000000000000000000000)))
     }
   }
+
+  uint256 constant UINT256_MAX = 2**256 - 1;
+  uint256 constant SUBMITTER_ONLY_WINDOW = 60 * 10;
+  uint256 constant Y = UINT256_MAX / SUBMITTER_ONLY_WINDOW;
+
+  function responsePossible(DisputeStages stage, uint256 since) internal view returns (bool) {
+    if (since > now) {
+      return false;
+    }
+
+    uint256 delta = now - since;
```

Comment:
Reading through this and I'm still getting confused around which way the window works. One one level it seems like it should be `>=`, i.e. more restrictive when the delta is large, but we have the opposite. Can we rename `delta` to `responseTimeRemaining` or something? `since` could also be clearer.

Reply:
Rearranged this a bit, that hopefully makes it clearer to the reader.